core/src/main/java/org/elasticsearch/index/query/WrapperQueryParser.java

```diff
@@ -56,11 +56,11 @@ public Query parse(QueryParseContext parseContext) throws IOException, QueryPars
 
         byte[] querySource = parser.binaryValue();
         try (XContentParser qSourceParser = XContentFactory.xContent(querySource).createParser(querySource)) {
-            final QueryParseContext context = new QueryParseContext(parseContext.index(), parseContext.indexQueryParserService());
+            final QueryCreationContext context = new QueryCreationContext(parseContext.index(), parseContext.indexQueryParserService());
```

Comment:
similar to the comment above, just use the QueryParseContext that we have here? We will deal with what needs to be done when we refactor this query?